benchmarks/spack/repo/packages/arepo/package.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+from spack.package import *
+
+class Arepo(MakefilePackage):
+    """Arepo is a massively parallel gravity and hydrodynamics code for astrophysical simulations."""
+    homepage = "https://gitlab.mpcdf.mpg.de/vrs/arepo.git"
+    git = "https://gitlab.mpcdf.mpg.de/vrs/arepo.git"
+    
+    
+    version("master", branch="master")
+   
+
+    depends_on('mpi')
+    depends_on('gsl')
+    depends_on('hdf5')
+
+    variant('fftw', default=False, description='FFTW support')
+    variant('hdf5', default=True, description='HDF5 support')
+    variant('hwloc', default=False, description='HWLOC support')
+        
+    depends_on('fftw',  when='+fftw')
+    depends_on('hwloc', when='+hwloc')
+    depends_on("mpich", when='^mpich')
+    depends_on("openmpi", when='^openmpi')
```

Comment:
Bit of a silly question, but isn't spack smart enough to figure this out on its own?

Reply:
Thanks for your suggestion! I have updated the Spack recipe to simplify the MPI dependency handling